ContactList/ContactList/FileOperations.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace ContactList
+{
+    public class FileOperations
+    {
+        private readonly ContactBook _contactBook;
+        private Mutex fileMutex = new Mutex(false, "appMutex");
```

Comment:
If your class holds a reference to an unmanaged object or managed object that implements IDisposable, it is a good practice to implement IDisposable for your class.